src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1436,4 +1436,7 @@
   <data name="EventsShouldNotHaveBeforeOrAfterPrefixTitle" xml:space="preserve">
     <value>Events should not have 'Before' or 'After' prefix</value>
   </data>
+  <data name="DoNotIgnoreMethodResultsMessageUserDefinedMethod" xml:space="preserve">
+    <value>'{0}' calls '{1'} but does not use the value the method returns. This method is defined as a user-option. Use the result in a conditional statement, assign the result to a variable, or pass it as an argument to another method.</value>
```

Comment:
```suggestion
    <value>'{0}' calls '{1}' but does not use the value the method returns. This method is defined as a user-option. Use the result in a conditional statement, assign the result to a variable, or pass it as an argument to another method.</value>
```

Reply:
Wondering why the test framework did not fail the tests when 2 arguments are being provided for expected diagnostics?

Reply:
Yep that's weird.

Reply:
Ok so the test framework is not failing when extra arguments are provided. `{1'}` is not parsed properly and so not considered as a placeholder. The `.WithArguments()` can take more arguments that required and won't fail. Might be worth improving this behavior.

Reply:
@sharwell for the suggestion. Possibly file a suggestion issue on roslyn-sdk repo for the testing library?

Reply:
https://github.com/dotnet/roslyn-sdk/issues/516